_studio/shared/umc/codec/h265_dec/src/umc_h265_task_supplier.cpp

```diff
@@ -1703,6 +1725,11 @@ UMC::Status TaskSupplier_H265::AddOneFrame(UMC::MediaData * pSource)
 
         UMC::MediaDataEx::_MediaDataEx* pMediaDataEx = nalUnit->GetExData();
 
+#if (MFX_VERSION >= 1025)
+    UMC::MediaData::AuxInfo* aux = (pSource) ? pSource->GetAuxInfo(MFX_EXTBUFF_DECODE_ERROR_REPORT) : NULL;
```

Comment:
code alignment here doesn't match to previous lines

Reply:
Ok, will align that.